### PR TITLE
Tests, fixes according to feedback document

### PR DESCRIPTION
**Pull request by mkawalek**
* notify, webhooks, api limits, endpoints tests.
* node version fix,
* manifest updated,
* hull traits are send as they are, no mapping required

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,82 +1,92 @@
 {
   "name": "Hull-customerio",
-  "description": "Sync users within customer.io service",
-  "tags": ["outgoing", "batch"],
+  "description": "Sync users with Customer.io",
   "picture": "picture.png",
-  "ui": false,
-  "admin": "admin.html",
```
Comment (mkawalek) [inline manifest.json]: I think that this is needed .. @michaloo could you plase confirm ?

Comment (michaloo) [inline manifest.json]: you mean the `admin` property?

Comment (mkawalek) [inline manifest.json]: yes :)

Comment (michaloo) [inline manifest.json]: yes, I think you need it.

**Pull request merged**